web/.env.development

```diff
@@ -0,0 +1 @@
+REACT_APP_BACKEND_URL=https://cors-anywhere.herokuapp.com/https://api-dot-gpay-group-buy.an.r.appspot.com/
```

Comment:
Can we use SERVER_URL instead? just to standardise the use of 'web' and 'server' instead of 'frontend' and 'backend'?

Reply:
hahaha same same to me sia but suree